src/coreclr/inc/corinfo.h

```diff
@@ -1723,6 +1725,17 @@ struct CORINFO_FIELD_INFO
     CORINFO_CONST_LOOKUP    fieldLookup;        // Used by Ready-to-Run
 };
 
+//----------------------------------------------------------------------------
+// getThreadLocalStaticBlocksInfo and CORINFO_THREAD_STATIC_BLOCKS_INFO: The EE instructs the JIT about how to access a thread local field
+
+struct CORINFO_THREAD_STATIC_BLOCKS_INFO
+{
+    uint32_t tlsIndex;
```

Comment:
I believe all the fields other than `offsetOfThreadLocalStoragePointer` here should be `CORINFO_CONST_LOOKUP`

Reply:
`tlsIndex` should be `CORINFO_CONST_LOOKUP`. I think the offsets can stay as `uint32_t`, for now at least.